python/bin/pmtiles-convert

```diff
@@ -32,56 +24,16 @@ if args.overwrite:
     elif os.path.isdir(args.output):
         shutil.rmtree(args.output)
 
-if args.input.endswith('.mbtiles') and args.output.endswith('.pmtiles'):
-    conn = sqlite3.connect(args.input)
-    cursor = conn.cursor()
-
-    with write(args.output) as writer:
-        for row in cursor.execute('SELECT zoom_level,tile_column,tile_row,tile_data FROM tiles WHERE zoom_level <= ? ORDER BY zoom_level,tile_column,tile_row ASC',(args.maxzoom or 99,)):
-            flipped = (1 << row[0]) - 1 - row[2]
-            writer.write_tile(row[0],row[1],flipped,row[3])
+kw = args.__dict__
```

Comment:
would prefer not having to change from the `args` object into a generic dict here!